cnd/src/swap_protocols/rfc003/create_swap.rs

```diff
@@ -289,34 +305,34 @@ impl<AL: Ledger, BL: Ledger, AA: Asset, BA: Asset> OngoingSwap<AL, BL, AA, BA> {
 }
 
 #[derive(Debug, Clone, PartialEq, strum_macros::Display)]
-pub enum SwapEvent<AL, BL, AA, BA>
+pub enum SwapEvent<AH, AT, BH, BT, AA, BA>
```

Comment:
This looks ugly but should ultimately get better as soon as it is going to be split into two!